scRNA-seq-advanced/04-differential_expression.Rmd

```diff
@@ -436,15 +438,14 @@ dim(filter_pb_sce)
 It looks like we only got rid of one group.
 We can do a quick check to see which group was removed by finding which column is no longer present in the filtered object.
 
-```{r removed columns}
+```{r removed columns, live=TRUE}
```

Comment:
I might not make this live but instead swap over the next-up `filter celltype` chunk to become live? What do you think?

Reply:
I think my original reason for not making the next chunk live is to make sure that we try and avoid trainee spelling mistakes for `Tumor_Myoblasts` when filtering. I'm trying to limit the smaller mistakes and having empty SCE objects going into DE.